lib/clients/backbeat-2017-07-01.api.json

```diff
@@ -194,7 +194,7 @@
         "DeleteObjectFromExpiration": {
             "http": {
                 "method": "DELETE",
-                "requestUri": "/_/backbeat/expiration/{Bucket}/{Key}"
+                "requestUri": "/_/backbeat/expiration/{Bucket}/{Key+}"
```

Comment:
(note: I don't have the full picture here, just asking to fully understand)
Should we exclude the `?` in case we have query parameters (like the versionID, or anything actually, so we are robust)?